forge-gui/res/cardsfolder/upcoming/nelly_borca_impulsive_accuser.txt

```diff
@@ -0,0 +1,11 @@
+Name:Nelly Borca, Impulsive Accuser
+ManaCost:2 R W
+Types:Legendary Creature Human Detective
+PT:2/4
+K:Vigilance
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigSuspect | TriggerDescription$ Whenever CARDNAME attacks, suspect target creature. Then goad all suspected creatures. (A suspected creature has menace and can't block.)
+SVar:TrigSuspect:DB$ AlterAttribute | ValidTgts$ Creature | Attributes$ Suspected | SubAbility$ DBGoad
+SVar:DBGoad:DB$ Goad | Defined$ Valid Creature.IsSuspected
+T:Mode$ DamageDoneOnce | CombatDamage$ True | ValidSource$ Creature.OppCtrl | ValidTarget$ Opponent | Execute$ TrigDraw | TriggerZones$ Battlefield | TriggerDescription$ Whenever one or more creatures an opponent controls deal combat damage to one or more of your opponents, you and the controller of those creatures each draw a card.
```

Comment:
wrong TriggerType

Reply:
Is Norn's Decree also wrong, then? What's the correct trigger type? All these damage triggers are inscrutable to me

Reply:
that one is only "one or more" once
check others with DamageAll